modules/era/sql/era_spell_list.sql

```diff
@@ -0,0 +1,6 @@
+LOCK TABLES `spell_list` WRITE;
+
+ALTER TABLE `spell_List`
+    ADD COLUMN IF NOT EXISTS `spell_radius_type` tinyint(1) unsigned NOT NULL DEFAULT 0 AFTER `spell_radius`;
+
+UNLOCK TABLES;
```

Comment:
There is another file I've brought back in from the back dated SQL folder with all of the in era spell adjustments if we can move this to that file.

Reply:
If you fixed that one up we are good, issue was that it was a TR one that messed up a bunch of stuff originally (hence why it was in that folder). I'm moving it to the regular SQL folder and added that column.

Reply:
@CriticalXI please review and let me know if this went into the correct sql nodule.

Reply:
I went through and added my own commit to this branch to update that table with my changes as well